tests/lib/pkgdb.sh

```diff
@@ -172,7 +172,7 @@ distro_install_local_package() {
             apt install $flags "$@"
             ;;
         fedora-*)
-            quiet dnf -y install "$@"
+            quiet dnf -y install --setopt=install_weak_deps=False "$@"
```

Comment:
I wonder if this will impact anything that only worked because of a weak dependency that was installed before. Let's see.